src/main/java/org/sagebionetworks/bridge/sdk/Config.java

```diff
@@ -500,6 +544,20 @@ private String val(Props prop) {
         checkNotNull(value, "The property '" + prop.getPropertyName() + "' has not been set.");
         return value.trim();
     }
+    
+    private String val(Props prop, NameValuePair queryParam) {
+        return val(prop, Lists.newArrayList(queryParam));
+    }
+    
+    private String val(Props prop, List<NameValuePair> queryParams) {
+        try {
+            URIBuilder builder = new URIBuilder(val(prop));
+            builder.addParameters(queryParams);
+            return builder.build().toString();
```

Comment:
How certain are we that URIBuilder will URL encode query params for us? Should we write a simple unit test to verify?